src/main/kotlin/com/matthewrussell/trwav/MarkerListJacksonUtils.kt

```diff
@@ -11,7 +11,10 @@ class MarkerListDeserializer : JsonDeserializer<List<CuePoint>>() {
         val node = p?.codec?.readTree<JsonNode>(p) ?: throw ParseException("Unable to parse json", 0)
         val cuePoints = mutableListOf<CuePoint>()
         for (key in node.fieldNames()) {
-            cuePoints.add(CuePoint(node.get(key).asInt(), key))
+            val labelNum = key
+                .replace("""[^0-9]""".toRegex(), "") // remove all letters
+                .replace("""^[0]+""".toRegex(), "") // remove leading zeroes
```

Comment:
this should be a util function instead of repeating the code below

Reply:
@jsarabia Moved to Utils class